### PR TITLE
completed javascript-conditionals

### PR DESCRIPTION
**Pull request by kyle-foral**
image

### CODE REVIEW

File: javascript-conditionals/javascript-conditionals-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# javascript-conditionals-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- Give 6 examples of comparison operators.
+Greater than >, Less than <, Equal to ==, Stricly equal to ===, Not equal to !=, Less than or equal to <=
+- What data type do comparison expressions evaluate to?
+Values or Variables
```
Comment (shawnkost) [inline javascript-conditionals/javascript-conditionals-quiz-notes.md:10]: I would adjust this answer. Think of primitive data types. (String, number, etc)

File: javascript-conditionals/main.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+function startsWithJ(string) {
+  if (string[0] === 'J') {
+    return true;
+  } else { return false; }
+}
+console.log('Starts with J ', startsWithJ('Cat'));
+
+function isOldEnoughToDrink(person) {
+  const Brad = {
```
Comment (shawnkost) [inline javascript-conditionals/main.js:39]: Variables should start with lowercase!

File: javascript-conditionals/main.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+  } else {
+    return false;
+  }
+}
+console.log('Is Even? ', isEven(6));
+
+function startsWithJ(string) {
+  if (string[0] === 'J') {
+    return true;
+  } else { return false; }
```
Comment (shawnkost) [inline javascript-conditionals/main.js:34]: I would recommend moving this to a new line, it's a lot more readable.

```
function startsWithJ(string) {
  if (string[0] === 'J') {
    return true;
  } else { 
    return false; 
  }
  ```

File: javascript-conditionals/main.js [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+}
+console.log('Starts with J ', startsWithJ('Cat'));
+
+function isOldEnoughToDrink(person) {
+  const Brad = {
+    age: 25,
+    name: 'Brad Long'
+  };
+  const over = Brad.age;
+  if (over > 21) {
```
Comment (shawnkost) [inline javascript-conditionals/main.js:44]: What happens if Brad is exactly 21?

File: javascript-conditionals/main.js [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+    return true;
+  } else { return false; }
+}
+console.log('Is person old enough to drink? ', isOldEnoughToDrink());
+
+function isOldEnoughToDrive(person) {
+  const Homer = {
```
Comment (shawnkost) [inline javascript-conditionals/main.js:51]: Variables should start with lowercase

File: javascript-conditionals/main.js [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+    name: 'Homer Simpson',
+    age: 30
+  };
+  const old = Homer.age;
+  if (old > 16) {
```
Comment (shawnkost) [inline javascript-conditionals/main.js:56]: What happens if Homer is exactly 16?

File: javascript-conditionals/main.js [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+
+function isOldEnoughToDrinkAndDrive(person) {
+  const Homer = {
+    name: 'Homer Simpson',
+    age: 1
+  };
+  const nogo = Homer.age;
+  if (nogo < 10) {
+    return false;
+  } else { return true; }
```
Comment (shawnkost) [inline javascript-conditionals/main.js:70]: You should never drink and drive 👮🏻‍♂️ 🚓 ! So this should only return `false`

File: javascript-conditionals/main.js [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+}
+console.log('Old enough to drink and drive? ', isOldEnoughToDrinkAndDrive());
+
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'neutral';
+  } else if (pH < 7 && pH > 0) {
```
Comment (shawnkost) [inline javascript-conditionals/main.js:77]: What happens if `pH` is exactly 0? That would still be considered acid

File: javascript-conditionals/main.js [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+    return 'acid';
+  } else if (pH > 7 && pH < 14) {
```
Comment (shawnkost) [inline javascript-conditionals/main.js:79]: What happens if `pH` is exactly 14? That would be considered base

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes requested here Kyle!

File: javascript-conditionals/javascript-conditionals-quiz-notes.md (hunk 1 above)
Comment (kyle-foral) [reply javascript-conditionals/javascript-conditionals-quiz-notes.md:10]: Are you saying I should adjust "Give 6 examples of comparison operators" or "What data type do comparison expressions evaluate to?"

File: javascript-conditionals/javascript-conditionals-quiz-notes.md [hunk 10]
```diff
@@ -7,7 +7,7 @@ After completing this exercise, you should be able to discuss or answer the foll
 - Give 6 examples of comparison operators.
 Greater than >, Less than <, Equal to ==, Stricly equal to ===, Not equal to !=, Less than or equal to <=
 - What data type do comparison expressions evaluate to?
-Values or Variables
+Strings, numbers, undefined, null, boolean, etc.
```
Comment (shawnkost) [inline javascript-conditionals/javascript-conditionals-quiz-notes.md:10]: It only evaluates to one of these data types!

File: javascript-conditionals/main.js [hunk 11]
```diff
@@ -36,47 +36,51 @@ function startsWithJ(string) {
...
 function isOldEnoughToDrinkAndDrive(person) {
-  const Homer = {
-    name: 'Homer Simpson',
+  const homer = {
+    name: 'homer Simpson',
     age: 1
   };
-  const nogo = Homer.age;
+  const nogo = homer.age;
   if (nogo < 10) {
```
Comment (shawnkost) [inline javascript-conditionals/main.js:70]: If you're 10, you wouldn't be able to either drink or drive. You should be checking for 2 separate numbers here.

**Review by shawnkost — CHANGES_REQUESTED** [review]
Hey Kyle! Sorry I missed one of your comments, I don't get notified when you leave comments on pull requests, so make sure any questions come to me via slack. A couple more changes requested here!

File: javascript-conditionals/javascript-conditionals-quiz-notes.md [hunk 12]
```diff
@@ -7,7 +7,7 @@ After completing this exercise, you should be able to discuss or answer the foll
 - Give 6 examples of comparison operators.
 Greater than >, Less than <, Equal to ==, Stricly equal to ===, Not equal to !=, Less than or equal to <=
 - What data type do comparison expressions evaluate to?
-Strings, numbers, undefined, null, boolean, etc.
+Numbers.
```
Comment (shawnkost) [inline javascript-conditionals/javascript-conditionals-quiz-notes.md:10]: Nope! If you do `if (age <= 16)` what would that evaluate to?

**Review by shawnkost — CHANGES_REQUESTED** [review]
One more change!

File: javascript-conditionals/javascript-conditionals-quiz-notes.md [hunk 13]
```diff
@@ -7,7 +7,7 @@ After completing this exercise, you should be able to discuss or answer the foll
 - Give 6 examples of comparison operators.
 Greater than >, Less than <, Equal to ==, Stricly equal to ===, Not equal to !=, Less than or equal to <=
 - What data type do comparison expressions evaluate to?
-Numbers.
+Numbers, strings, values and variables.
```
Comment (shawnkost) [inline javascript-conditionals/javascript-conditionals-quiz-notes.md:10]: Still nope! It only evaluates to one data type, not multiple. Btw values and variables aren't a data type.

In the image below, what does it say condition will evaluate to?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else

Screenshot 2023-03-22 at 4 14 03 PM

**Review by shawnkost — CHANGES_REQUESTED** [review]
Almost there!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**